File: VisitBosnia/VisitBosnia.Model/SearchObjects/PostReplySearchObject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VisitBosnia.Model.SearchObjects
{
    public class PostReplySearchObject
    {
        public int? AppUserId { get; set; }
        public int? PostId { get; set; }
        public bool IncludeAppUser { get; set; }
    }
}

File: VisitBosnia/visit_bosnia_mobile/lib/model/postReply/post_reply_search_object.dart
class PostReplySearchObject {
  int? appUserId;
  int? postId;
  bool? includeAppUser;

  PostReplySearchObject({this.appUserId, this.postId, this.includeAppUser});

  PostReplySearchObject.fromJson(Map<String, dynamic> json) {
    appUserId = json['appUserId'];
    postId = json['postId'];
    includeAppUser = json['includeAppUser'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['appUserId'] = this.appUserId;
    data['postId'] = this.postId;
    data['includeAppUser'] = this.includeAppUser;
    return data;
  }
}
